default_validator/src/lib.rs

```diff
@@ -3,14 +3,164 @@
 #![no_std]
 #![allow(missing_docs, clippy::missing_errors_doc)]
 
+extern crate alloc;
 #[cfg(not(test))]
 extern crate panic_halt;
 
-use iroha_validator::prelude::*;
+use alloc::borrow::ToOwned as _;
+
+use iroha_validator::{
+    data_model::evaluate::ExpressionEvaluator, default::default_permission_token_schema,
+    iroha_wasm, prelude::*,
+};
+
+/// Validator that replaces some of [`Validate`]'s methods with sensible defaults
+///
+/// # Warning
+///
+/// The defaults are not guaranteed to be stable.
+#[derive(Debug, Clone)]
+pub struct Validator {
+    verdict: Result,
+    block_height: u64,
+    host: iroha_wasm::Host,
+}
+
+impl Validator {
+    /// Construct [`Self`]
+    pub fn new(block_height: u64) -> Self {
+        Self {
+            verdict: Ok(()),
+            block_height,
+            host: iroha_wasm::Host,
+        }
+    }
+
+    fn ensure_genesis(block_height: u64) -> MigrationResult {
+        if block_height != 0 {
+            return Err("Default Validator is intended to be used only in genesis. \
```

Comment:
As a slight recommendation, o'd wager this doesn't need to be represented as a string, but rather there ought to be an enum for this

Reply:
I'm not entirely convinced because this is just an example validator. And there is no way of enforcing that every 1st validator will implement this check

Reply:
It doesn't need to.just like not every http response is not emitted by every server

Reply:
created #3860